package br.com.digitalhouse.aula8

fun main(){
// deve dar uma exception, a partir daqui
/*
    val numeros = listOf("um", "dois", "tres")
    println("Terceiro elemento: ${numeros.get(3)}")
*/
/*
    var x = 2
    var y = 0
    var z = 6

    println((x * z)/y)
*/



}